src/vs/workbench/contrib/terminal/browser/terminal.ts

```diff
@@ -593,7 +594,7 @@ export interface ITerminalInstance {
 	/**
 	 * The xterm.js instance for this terminal.
 	 */
-	readonly xterm?: IXtermTerminal;
+	readonly xterm?: XtermTerminal;
```

Comment:
What did you need from here? I was thinking you would want to use a ITermianlContribution so an a11y buffer exists for all terminals